### PR TITLE
Fixed broken unicode characters after spawn in buffers

### PR DESCRIPTION
**Pull request by bragovo**

### CODE REVIEW

**Comment by rhys-vdw** [comment]
Looks good! So glad you found a solution. One question...

Can it work like this?

```js
  var dataBuffers = []
  child.stdout.on('data', function (data) {
    dataBuffers.push(data)
  })
  // ...
  var matches = Buffer.concat(dataBuffers).toString('utf8').match(sourceRegex)
```

I think calling `Buffer.concat` does unnecessary copies while the output is being read.

**Comment by rhys-vdw** [comment]
Also, do you have any thought on how we might add test coverage? I'm happy to merge without it, but it would be good to prevent regressions in future.

File: index.js [hunk 1]
```diff
@@ -89,11 +89,11 @@ function transformSource (runner, config, source, map, callback) {
     ? setTimeout(function () { child.kill() }, config.timeoutMs)
     : -1
 
-  var dataBuffers = []
+  var dataBuffers = Buffer.alloc(0)
   child.stdout.on('data', function (data) {
     // Output is delimited to filter out unwanted warnings or other output
```
Comment (rhys-vdw) [inline index.js]: Can you move comment down to the line where `new RegExp` is called pls?

Comment (bragovo) [reply index.js]:
```suggestion
    // Output is delimited to filter out unwanted warnings or other output
```
This one? Sure.

**Comment by rhys-vdw** [comment]
Beautiful 👌

**Comment by bragovo** [comment]
Yes, you right, it's better to concat only once. Actually I don't know right now how to test this, but when I have some time I'll think about it.

**Pull request merged**